provider/pkg/await/await.go

```diff
@@ -162,6 +163,8 @@ func Creation(c CreateConfig) (*unstructured.Unstructured, error) {
 	// nolint
 	// https://github.com/kubernetes/kubernetes/blob/54889d581a35acf940d52a8a384cccaa0b597ddc/pkg/kubectl/cmd/apply/apply.go#L94
 
+	patchResource := kinds.PatchQualifiedTypes.Has(c.URN.QualifiedType().String())
```

Comment:
nit: I'd call this `isPatchResource` since patch can be either a noun or a verb.